src/eval_tool.py

```diff
@@ -0,0 +1,64 @@
+import cPickle
+import os
+import pandas as pd
+import numpy as np
+
+from keras.models import load_model
+from keras.preprocessing.text import Tokenizer
+from keras.preprocessing.sequence import pad_sequences
+from keras.utils import to_categorical
+from sklearn import metrics
+
+def compute_auc(y_true, y_pred):
+    fpr, tpr, thresholds = metrics.roc_curve(y_true, y_pred)
+    return metrics.auc(fpr, tpr)
+
+class EvalTool():
```

Comment:
What do you think about naming this "Model"?

Reply:
good idea. Done.